src/reducers/postReducer.js

```diff
@@ -1,14 +1,17 @@
 import initialState from './initialState';
 import { SampleDataService } from '../services/sampledata-service';
+import {DataService} from "../services/data-service";
 import { GET_POSTS, SAVE_POST, GET_EXAMPLE_POSTS } from '../actions/actionTypes'
 import { Post } from '../models/post'
 
 export default function postReducer(state = initialState.posts, action) {
 	// TODO: change this to posts service
 	var sampleDataService = new SampleDataService(9);
+	var dataService = new DataService(9);
 	switch(action.type) {
 		case GET_POSTS:
-			return state;
+		var postsFromDB = dataService.getPostsFromDB()
```

Comment:
This might be nitpicky but while we're here indent please :)

Reply:
yessir.